### DISCUSSION TITLE
mitiimweb dont sync mitimproxy: Connection lost error code 1006

### DISCUSSION THREAD

**Discussion by reinaldoviedo94** [mitmproxy/mitmproxy#6451] [Q&A] [answered]
Hello i runnig your awesome tool, following docker installation method, but when open mitmweb on browser i dont see event logs in page.

![Captura desde 2023-11-04 02-24-45](https://github.com/mitmproxy/mitmproxy/assets/82004527/c7d443bd-cd1f-4765-b7c0-cca967e5b98a)

**Comment by mhils** [comment, marked as answer]
See https://github.com/mitmproxy/mitmproxy/issues/6452 :)

**Reply by reinaldoviedo94** [reply]
thanks
